Edits to partnerships guidance (#1583)
DFE-Digital/register-early-career-teachers-public: app/views/api/guidance/guidance_for_lead_providers/create_view_and_update_partnerships.md

4. Lead providers find schools they want to partner with using `GET /schools` API endpoint. Schools that have selected them will show as `expression_of_interest = true`. 
5. Lead providers create partnerships via the `POST /partnerships` endpoint. They’ll then see any participants the school has assigned to them in `GET /participants`.

> Lead providers can create a partnership with a school as soon as the cohort opens, regardless of if the school has registered any ECTs or mentors with them yet. Schools can also choose to reuse partnerships from previous cohorts so lead providers do not need to create a new partnership.

Reviewer: Suggested tweak to the second line: Schools can reuse partnerships from previous cohorts, so lead providers don’t need to create new ones.

Author: good shout

> Lead providers can create a partnership with a school as soon as the cohort opens, regardless of if the school has registered any ECTs or mentors with them yet. Schools can also choose to reuse partnerships from previous cohorts so lead providers do not need to create a new partnership.

Author: ```suggestion
Lead providers can create a partnership with a school as soon as the cohort opens, regardless of if the school has registered any ECTs or mentors with them yet.  Schools can also reuse partnerships from previous cohorts, so lead providers don’t need to create new ones.

```

4. Lead providers find schools they want to partner with using `GET /schools` API endpoint. Schools that have selected them will show as `expression_of_interest = true`. 
5. Lead providers create partnerships via the `POST /partnerships` endpoint. They’ll then see any participants the school has assigned to them in `GET /participants`.

> Lead providers can create a partnership with a school as soon as the cohort opens, regardless of if the school has registered any ECTs or mentors with them yet.  Schools can also reuse partnerships from previous cohorts, so lead providers don’t need to create new ones.

Reviewer 2: ```suggestion
Lead providers can create a partnership with a school as soon as the cohort opens, regardless of if the school has registered any ECTs or mentors with them yet. Schools can also reuse partnerships from previous cohorts, so lead providers don’t need to create new ones.
```

Is the double space here intentional?
